.circleci/config.yml

```diff
@@ -15,13 +15,13 @@ executors:
 # Job definitions
 jobs:
   style:
-    executor: clojure
+    executor: clojure-java-11
```

Comment:
If the combinatorial versions start getting out of hand, we should look into https://circleci.com/docs/2.0/dynamic-config/